web/core/store/issue/issue-details/attachment.store.ts

```
@@ -104,21 +126,56 @@ export class IssueAttachmentStore implements IIssueAttachmentStore {
+        projectId,
+        issueId,
+        file,
+        (progressEvent) => {
+          const progressPercentage = Number(((progressEvent.progress ?? 0) * 100).toFixed(0));
+          this.debouncedUpdateProgress(issueId, tempId, progressPercentage);
+        }
+      );
+      const issueAttachmentsCount = this.getAttachmentsByIssueId(issueId)?.length ?? 0;
 
-    return response;
+      if (response && response.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)